locust/test/test_main.py

```diff
@@ -509,6 +798,7 @@ def tick(self):
                 stdout=PIPE,
                 stderr=PIPE,
                 text=True,
+                env=os.environ.copy(),
```

Comment:
this is essentially a no-op :) subprocess always inherits a copy of the current environ :)